boards/arm/cy8cproto_063_ble/cy8cproto_063_ble.yaml

```diff
@@ -18,3 +18,4 @@ supported:
   - uart
```

Comment:
Change the commit title for something more explicit. "Initial version"... of what?
And in the message, add more details.